rr/rr_policies.md

```diff
@@ -1,5 +1,19 @@
 # Registered Reports - Journal Policies
 
+## Timelines
+In general, one year is the expected time between Phase 1 submission and Phase 2 submission. Each conference will outline the specific dates. Reviewer continuity becomes challenging beyond this point.
```

Comment:
Expected sounds like it the average or median. But you mean more like a max? (given the later comment about reviewer continuity)

Reply:
right - but I'm inclined to just point to guidance on RR from rr.peercommunityin.org. Here is what they say:
> Should Stage 1 IPA be forthcoming, authors will be asked to provide PCI RR with an estimated submission date for the completed Stage 2 manuscript. This deadline can be readily altered in consultation with the recommender (e.g. in case of delays requiring additional time to complete the research). However, in the event that the authors (a) fail to submit the Stage 2 manuscript within 6 months of the mutually agreed deadline, while also (b) becoming non-responsive during this period to enquiries, then the manuscript will be considered by PCI RR to be withdrawn.

Reply:
Sounds good to me.